js/views/conversation_view.js

```diff
@@ -358,7 +358,11 @@
 
             if (!this.isHidden() && !window.isFocused()) {
                 // The conversation is visible, but window is not focused
-                this.updateLastSeenIndicator({scroll: false});
+                if (!this.lastSeenIndicator) {
+                    this.resetLastSeenIndicator({scroll: false});
+                } else if (this.view.atBottom() && this.model.get('unreadCount') === this.lastSeenIndicator.getCount()) {
```

Comment:
If we're at the bottom, and the count has not changed, scroll into view... Not sure what issue you're trying to address here.